lib/Runtime/Library/InJavascript/Intl.js

```diff
@@ -7,13 +7,23 @@
 // Core intl lib
 (function (EngineInterface, InitType) {
     var platform = EngineInterface.Intl;
+
+    // allow unit tests to disable caching behavior for testing convenience but have this always `true` in real scenarios
+    platform.useCaches = true;
```

Comment:
I don't see how test code can set this?

Reply:
It is used in the "IntlPlatformTests" chakrafull unit test

Reply:
Through the Debug object (exposed through a test flag) which allows tests to manipulate the EngineInterfaceObject passed to Intl code. At some point we should migrate that test option and associated tests to ChakraCore.